msteams-platform/concepts/design/overview-analytics.md

```diff
@@ -112,27 +112,27 @@ After you extract the Teams-specific information from the tab context, some poss
 | Track Teams-specific information | Outcome |
 | --- | --- |
 | User's organization. | Track demand for your app, especially if you offer trial period for new sign-ups or freemium offering for your SaaS product for sales outreach, onboard organization, check in for assistance. |
-| User’s organization type and user’s role. | Use license type and tenant SKU to construct a customer profile for the Teams user. Slice your analytics data, such as, usage, sessions, retention, and so on. accordingly, and tailor your features, experiences, and so on. accordingly. |
+| User’s organization type and user’s role. | Use license type and tenant SKU to construct a customer profile for the Teams user. Slice your analytics data, such as, usage, sessions, and retention and tailor your features, experiences accordingly. |
```

Comment:
and retention, and tailor your features and experiences accordingly.

Reply:
Updated